client/my-sites/site-settings/form-analytics-stores.jsx

```diff
@@ -53,13 +70,53 @@ class FormAnalyticsStores extends Component {
 		return this.renderSettings( settings, disableAll );
 	};
 
+	renderEnhancedSettings = disableAll => {
+		const { translate } = this.props;
+		const settings = [
+			{
+				key: 'enh_ec_tracking',
+				label: translate( 'Enable Enhanced eCommerce' ),
+				explanation: translate(
+					'Before enabling, turn on enhanced eCommerce in your Google Analytics dashboard.'
+				),
+				children: [
+					{
+						key: 'enh_ec_track_remove_from_cart',
+						label: translate( 'Remove from Cart events' ),
+					},
+					{
+						key: 'enh_ec_track_prod_impression',
+						label: translate( 'Product impressions from listing pages' ),
+					},
+					{
+						key: 'enh_ec_track_prod_click',
+						label: translate( 'Product clicks from listing pages' ),
+					},
+					{
+						key: 'enh_ec_track_prod_detail_view',
+						label: translate( 'Product detail views' ),
+					},
+					{
+						key: 'enh_ec_track_checkout_started',
+						label: translate( 'Checkout process initiated' ),
+					},
+				],
+			},
+		];
+
+		return this.renderSettings( settings, disableAll );
+	};
+
 	render = () => {
 		const { disabled, translate } = this.props;
+		const showEnhanced = config.isEnabled( 'jetpack/google-analytics-for-stores-enhanced' );
 
 		return (
 			<div>
 				<FormLegend>{ translate( 'Basic store analytics' ) }</FormLegend>
 				{ this.renderBasicSettings( disabled ) }
+				{ showEnhanced && <FormLegend>{ translate( 'Enhanced store analytics' ) }</FormLegend> }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Basic store analytics' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).